### ISSUE TITLE
libtensorflow_compression.so not found

### ISSUE THREAD

**Issue by michaelshiyu** [tensorflow/compression#21]
Hi, when I try to import tensorflow_compression, I get the following error:

`tensorflow.python.framework.errors_impl.NotFoundError: /home/user/Github/compression/tensorflow_compression/python/ops/../../cc/libtensorflow_compression.so: cannot open shared object file: No such file or directory`

I searched system-wise but still couldn't find a file with this name. Could anyone please help? Below are my settings:

```
python=3.4
tensorflow-gpu=1.13.1
tensorflow_compression=1.2
```

Many thanks.

**Comment by jonycgn** [comment]
Hi, could you try to install tensorflow 1.14, and tensorflow_compression 1.2b2?
I think this should fix your issue.

**Comment by michaelshiyu** [comment]
Hi @jonycgn, the issue is fixed by switching to tensorflow-gpu 1.14.0. Thanks a lot for the help.

**Comment by xinziXu** [comment]
I met the same problem, but I am using tensorflow-gpu 1.14.0 and tensorflow_compression 1.2b2
